nvflare/private/fed/client/client_app_runner.py

```diff
@@ -89,9 +97,17 @@ def create_run_manager(self, args, conf, federated_client, workspace):
         )
 
     def start_command_agent(self, args, client_runner, federated_client, fl_ctx):
+        start = time.time()
+        while federated_client.cell is None:
+            time.sleep(0.1)
+            if time.time() - start > 60.0:
+                raise RuntimeError("No cell created. Failed to start the command_agent for ClientAppRunner.")
```

Comment:
these randomly pick values should be configurable somewhere

Reply:
Added configurable.